.github/workflows/strict-null-checks.yml

```diff
@@ -0,0 +1,80 @@
+name: Strict Null Checks Burndown
+# this is a temporary workflow while we burn down the errors
+# from typescript's strictNullChecks option. Once we get to 0
+# we can turn that option on all the time and delete this job!
+
+on:
+  pull_request:
```

Comment:
I wrote these jobs in a separate file (instead of in `main.yml`) because we have that file running on

```yml
on:
  push:
    branches:
      - 'main'
      - 'v3.0.0-dev'
  pull_request:
    branches:
      - '**'
```

for this workflow I think we only want it to run on a PR, since we need to have a PR on which to write our little comment report out.

Reply:
👍 I like it!